src/Microsoft.DotNet.Interactive.CSharpProject/Commands/CompileProject.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Interactive.Commands;
+
+namespace Microsoft.DotNet.Interactive.CSharpProject.Commands;
+
+public class CompileProject : KernelCommand
+{
+    public string Code { get; }
```

Comment:
What is an example of this code? This command can be clearer in intent.

Reply:
not sure I understand. That code is anything user want to be compiled with the context of the project. I would argue that that command is not compileproject, is CompleCode. The fact that the project kernel is compiling that code against a project sounds like a n implementation detail of the proejct kernel.

Should this question be addressed in the pr @brettfo  is working on with the actual kernel implementation? This pr goal is to move things in their place. Your point is something I would like to address with @brettfo too

Reply:
If it's more correctly `CompileCode`, what is the context for the code? Is it to be inlined somehow into an existing project or file? Is it a complete program?